components/prize_pool/commons/prize_pool.lua

```diff
@@ -565,10 +566,21 @@ function PrizePool:_buildRows()
 			end
 
 			if opponentIsParty then
-				if opponent.opponentData.players[1] and opponent.opponentData.players[1].team then
-					row:addCell(TableCell{content = {tostring(OpponentDisplay.BlockOpponent{
-						opponent = {type = 'team', template = opponent.opponentData.players[1].team}
-					})}, css = opponentCss})
+				local playerTeams = mw.html.create('div')
+				local hasTeam
+				for _, player in ipairs(opponent.opponentData.players) do
+					if player.team then
+						hasTeam = true
+						playerTeams:node(OpponentDisplay.BlockOpponent{
+							opponent = {type = 'team', template = player.team}
+						})
+					else
+						playerTeams:node(EMPTY_TEAM)
```

Comment:
This is unused on the test page, are you sure it's needed?

Reply:
it is used:
![Screenshot 2022-08-05 10 17 19](https://user-images.githubusercontent.com/75081997/183034261-d64414fa-2c8d-443a-b514-bdd1196fe886.png)

Reply:
Those didn't show up when I looked at the page, was just a filler icon

Reply:
yeah i changed it but didn't purge the page

https://github.com/Liquipedia/Lua-Modules/pull/1623/commits/c0d65b08cbc36afad003f93576faee3a5e382753